services/web/server/src/simcore_service_webserver/catalog/_handlers.py

```diff
@@ -76,7 +77,9 @@ async def list_services(request: Request):
     # assert parse_obj_as(list[ServiceGet], data_array) is not None  # nosec
     #
 
-    return envelope_json_response(data_array)
+    return await asyncio.get_event_loop().run_in_executor(
+        None, envelope_json_response, data_array
+    )
```

Comment:
Q: would it not be better to transform envelope_json_response in an async one and move this code inside?

Reply:
maybe but that implies a lot of changes and potential issues. for now it's done where it causes problems.